src/coord/src/persistcfg.rs

```diff
@@ -197,14 +197,18 @@ impl PersisterWithConfig {
         }
     }
 
-    pub fn details(&self, id: GlobalId, pretty: &str) -> Result<Option<PersistDetails>, Error> {
+    pub fn details(
```

Comment:
let's call this table_details now (ditto for table_details_from_name). maybe for the stuff in catalog.rs too?